product-matrix.json

```diff
@@ -1,32 +1,5 @@
 {
   "list": [
-    {
-      "channel": "stable",
-      "comments": "IntelliJ 2020.1.1, Android Studio 4.1",
-      "name": "Android Studio 4.1",
-      "version": "4.1",
-      "ideaProduct": "android-studio",
-      "ideaVersion": "201.6565218",
-      "dartPluginVersion": "201.7223.99",
-      "sinceBuild": "201.7223",
-      "untilBuild": "201.*",
-      "filesToSkip": [
-        "src/io/flutter/utils/AndroidLocationProvider.java",
-        "flutter-studio/src/io/flutter/actions/FlutterNewProjectAction.java"
-      ]
-    },
-    {
```

Comment:
We can't delete this yet.

Reply:
This should only be used by 2020.1.1 / AS 4.1?

Reply:
Ah, you mean the section below.